deploy/openshift/hawkular-openshift-agent-configmap.yaml

```diff
@@ -11,11 +11,11 @@ data:
     kubernetes:
       tenant: ${POD:namespace_name}
     hawkular_server:
-      url: https://hawkular-metrics.openshift-infra.svc.cluster.local       
+      url: https://hawkular-metrics.openshift-infra.svc.cluster.local
       credentials:
         username: secret:openshift-infra/hawkular-metrics-account/hawkular-metrics.username
         password: secret:openshift-infra/hawkular-metrics-account/hawkular-metrics.password
-      ca_cert_file: secret:openshift-infra/hawkular-metrics-certificate/hawkular-metrics-ca.certificate
+      ca_cert_file: secret:openshift-infra/hawkular-metrics-certs/tls.truststore.crt
```

Comment:
The heapster pod in the `openshift-infra` project use secret `hawkular-metrics-certs/tls.crt` as CA file.
```
$ oc get rc heapster -n openshift-infra -o yaml | grep hawkular-metrics-certs/
--sink=hawkular:https://hawkular-metrics:443 ... &caCert=/hawkular-metrics-certs/tls.crt
```

Can you update the pull request to use this line?
```
ca_cert_file: secret:openshift-infra/hawkular-metrics-certs/tls.crt
```